김재관/index.js

```diff
@@ -0,0 +1,44 @@
+const btns = document.querySelectorAll('.js-btn');
+const inputString = document.querySelector('.js-input-string');
+const result = document.querySelector('.js-result');
+const btnAc = document.querySelector('.js-btn-ac');
+const btnResult = document.querySelector('.js-btn-result');
+const btnDe = document.querySelectorAll('.js-btn-de');
+const WARRING_MSG = '비활성화 된 버튼입니다.';
+
+const btnClickHandler = (e) => {
+    const { target } = e;
+    if (inputString.innerHTML === '0') {
+        inputString.innerHTML = '';
+        inputString.innerHTML += target.innerHTML;
+    } else {
+        inputString.innerHTML += target.innerHTML;
+    }
+};
```

Comment:
```suggestion
const btnClickHandler = (e) => {
    const { target } = e;
    if (inputString.innerHTML === '0') {
        inputString.innerHTML = '';
    }
    inputString.innerHTML += target.innerHTML;
};
```

조금 더 DRY하게 작성할 수도 있는 점 말씀드립니다 ~

Reply:
아하 감사합니다! 👍